.travis.yml

```diff
@@ -13,15 +17,23 @@ env:
 
 r_packages:
   - devtools
+  - Cairo # this was missing on OSX, no idea why this wasn't installed :-( Needed for repr
 
 r_github_packages:
   # we currently still need IRdisplay from github in all cases...
   - IRkernel/IRdisplay
 
 
 script:
-  - source .travis-build.sh
+  - export CHECK_LOG=${TRAVIS_BUILD_DIR}/IRkernel.Rcheck/00check.log
+  - export INSTALL_LOG=${TRAVIS_BUILD_DIR}/IRkernel.Rcheck/00install.out
+  - bash .travis-build.sh
   - grep -q -R 'WARNING' "$CHECK_LOG" ; [ $? -ne 0 ]
   # TODO: currently we do have 1 NOTE: Found the following assignments to the global environment...
   #- grep -q -R 'NOTE' "$CHECK_LOG"; [ $? -ne 0 ]
 
+after_failure:
+  - printf "travis_fold:start:show.log\n"
```

Comment:
1. use the travis_fold fucntion instead of manually printing stuff
2. this doesn’t work properly. it’s run as separate commands and messes up stuff. you need to extract this into a shell script.
3. are you sure those logs aren’t already in some folded section?